Api/Presentation/IncomesController.cs

```diff
@@ -1,47 +1,86 @@
 using Api.Domain;
-
+using Api.Domain.Services;
 using AutoMapper;
-
 using Microsoft.AspNetCore.Mvc;
 
 [ApiController]
 [Route("api/incomes")]
 public class IncomesController : ControllerBase
 {
     private readonly IMapper _mapper;
+    private readonly IIncomeService _incomeService;
 
-    public IncomesController(IMapper mapper)
+    public IncomesController(IMapper mapper, IIncomeService incomeService)
     {
         _mapper = mapper;
+        _incomeService = incomeService;
     }
 
     [HttpPost]
     public async Task<IActionResult> LogIncome([FromBody] CreateIncomeRequest createIncomeRequest)
```

Comment:
Add the default value "created at" attribute in the Income model, and please wait a bit to use the user id, otherwise there may be empty or invalid fields.